Translation guidelines (#735)
cuttle-cards/cuttle: docs/translation-guidelines.md

# Translation guidelines

Here, we try to determine a terminology for the game of scuttle that will help future translators.  The goal is to keep the 'Cuttle feel' and have a uniformed terminology throughout the translations.

## The "Cuttle feel"

> Some of the terminology refers to the seas and the ocean's depth, but it's not meant to be applied to every term in the game.  Most words used are just standard words used in card games in general. Each language should use words and expressions that feel right to native speakers and not use literal trnaslations with the only goal of using the same word/expression as in the English version.  

Reviewer: ```suggestion
Some of the terminology refers to the seas and the ocean's depth, but it's not meant to be applied to every term in the game.  Most words used are just standard words used in card games in general. Each language should use words and expressions that feel right to native speakers. We are not limited to literal translations for every term. The goal is to convey the meaning and feel.

```
